subprojects/parseq/src/main/java/com/linkedin/parseq/Task.java

```diff
@@ -893,6 +893,43 @@
     return withTimeout;
   }
 
+  /**
+   * Equivalent to {@code withDelay(null, time, unit)}.
+   * @see #withDelay(String, long, TimeUnit)
+   */
+  default Task<T> withDelay(final long time, final TimeUnit unit) {
+    return withDelay(null, time, unit);
+  }
+
+  /**
+   * Creates a new task whose execution is delayed.
```

Comment:
This comment is not quite accurate. This is actually delaying the execution of its wrapped task by the given period.

Reply:
Updated the doc.